Loop/Managers/ServicesManager.swift

```diff
@@ -201,6 +205,40 @@ extension ServicesManager: ServiceDelegate {
         log.default("Service with identifier '%{public}@' deleted", service.serviceIdentifier)
         removeActiveService(service)
     }
+    
+    func handleRemoteOverride(name: String, durationTime: TimeInterval?, remoteAddress: String) async throws {
+        try await remoteCommandHandler?.handleRemoteOverride(name: name, durationTime: durationTime, remoteAddress: remoteAddress)
+    }
+    
+    func handleRemoteOverrideCancel() async throws {
+        try await remoteCommandHandler?.handleRemoteOverrideCancel()
+    }
+    
```

Comment:
Intercepting these enactment errors to show these existing error notifications. This is not new in this PR but moved. These are not parsing/OTP errors but rather things specific to bolus/carbs (i.e. pump issue maybe)